src/components/Dropdowns/DropdownWithColorPicker.tsx

```diff
@@ -0,0 +1,73 @@
+import { Color as AntdColor } from "@rc-component/color-picker";
+import { ColorPicker } from "antd";
```

Comment:
It looks like the GitHub Action check is upset about the order of these imports?